src/pages/admin/student/edit/components/editStudentComponent.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { Student } from '../../../../../model/student';
+
+interface IProps {
+  student: Student;
+  updateStudent: (property: string, value: any) => void;
+  saveStudent: (event: any) => void;
+}
+
+function updateStudentFullName(e: any) {
+  this.props.updateStudent('fullname', e.target.value);
+}
+
+function updateStudentEmail(e: any) {
+  this.props.updateStudent('email', e.target.value);
+}
+
+function updateStudentIsActive(e: any) {
+  this.props.updateStudent('isActive', e.target.value);
+}
+
+function saveStudent(e: any) {
+  this.props.saveStudent(event);
+}
```

Comment:
This functions should leave inside the EditStudentComponent, does this works as it is? mmm I guess so because we are binding "this" specifically.